scm-core/src/main/java/sonia/scm/repository/spi/RemoteCommandRequest.java

```diff
@@ -21,140 +21,42 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-    
-package sonia.scm.repository.spi;
 
-//~--- non-JDK imports --------------------------------------------------------
+package sonia.scm.repository.spi;
 
-import com.google.common.base.MoreObjects;
-import com.google.common.base.Objects;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
 import sonia.scm.repository.Repository;
 
 import java.net.URL;
 
-//~--- JDK imports ------------------------------------------------------------
-
 /**
- *
  * @author Sebastian Sdorra
  * @since 1.31
  */
-public abstract class RemoteCommandRequest implements Resetable
-{
-
-  /**
-   * {@inheritDoc}
-   */
-  @Override
-  public boolean equals(Object obj)
-  {
-    if (obj == null)
-    {
-      return false;
-    }
+@Getter
```

Comment:
Yes, we got classloading errors because the plugin classloader could not access the webapp classloader fields for this class. I already discussed this with @pfeuffer.